data.ts

```diff
@@ -44,6 +44,11 @@ export async function populate(path: string, opts: ClientOptions) {
   const data = await getData(path)
   if (data) {
     const client = new Client(opts)
-    await client.bulk({ body: data })
+    const batch_size = 10
+    for (let i = 0; i < data.length; i += batch_size) {
+      const batch = data.slice(i, i + batch_size)
```

Comment:
- Use a [`for...of` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) or [`Array.prototype.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) rather than explicit indexing.
- Use [lodash's `chunk`](https://lodash.com/docs/4.17.15#chunk).